ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -589,6 +599,24 @@ export default defineComponent({
       localState.dataLoaded = true
     })
 
+    const mapTransferReviewValidations = () => {
+      const validationSections = [
+        'staff-transfer-submitting-party',
+        'transfer-ref-num-section',
+        'transfer-confirm-section',
+        'transfer-certify-section',
+        'staff-transfer-payment-section'
+      ]
+      const mappedSections = validationSections.map(function (key, val) {
+        return { section: key, value: localState.sectionValidationStates[val] }
+      })
+      if (!isRoleStaffReg.value) {
+        mappedSections.shift()
+        mappedSections.pop()
+      }
+      return mappedSections
+    }
```

Comment:
Should this be computed as well and include/return
`const invalidSection = document.getElementById(mappedSections.filter(item => item.value === false[0].section)`?

Then you could just pass this value (`getInvalidSection`) or whatever into your scrollToSection method?

Reply:
Sure, I'll do that to keep the methods completely separate